tests/js/app/testArticleHTMLRenderer.js

```diff
@@ -74,8 +74,8 @@ describe('Article HTML Renderer', function () {
         let html_no_title = renderer.render(wikibooks_model);
         renderer.show_title = true;
         let html_with_title = renderer.render(wikibooks_model);
-        expect(html_with_title).toMatch('Wikibooks title');
-        expect(html_no_title).not.toMatch('Wikibooks title');
+        expect(html_with_title).toMatch('<h1>Wikibooks title</h1>');
```

Comment:
Does this really need to be changed? `<h1>` is presentation information so I'd recommend keeping it out of here if possible.